### PR TITLE
Fix following Insimo PR on freemotion

### PR DESCRIPTION
**Pull request by EulalieCoevoet**

### CODE REVIEW

File: component/initSoftRobots.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ const char* getModuleComponentList()
 }
 
 SOFA_LINK_CLASS(AnimationEditor)
-SOFA_LINK_CLASS(CommunicationController)
+//SOFA_LINK_CLASS(CommunicationController)
```
Comment (damienmarchal) [inline component/initSoftRobots.cpp]: Commenting dead feature is a bad practice. Don't commit it. 
A small explanation why:
https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

File: component/constraint/model/CableModel.h [hunk 2]
```diff
@@ -31,9 +31,9 @@
 #ifndef SOFA_COMPONENT_CONSTRAINTSET_CABLEMODEL_H
 #define SOFA_COMPONENT_CONSTRAINTSET_CABLEMODEL_H
 
-#include <sofa/core/behavior/Constraint.h>
-#include <sofa/core/behavior/MechanicalState.h>
-#include <sofa/core/behavior/OdeSolver.h>
+//#include <sofa/core/behavior/Constraint.h>
```
Comment (damienmarchal) [inline component/constraint/model/CableModel.h]: Commenting dead feature is a bad practice. Don't commit it. 
A small explanation why:
https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

**Comment by damienmarchal** [comment]
Hello, thank for the PR. 

Bruno is reporting it is not yet compiling so I will give it a look.

**Comment by damienmarchal** [comment]
Let's merge it...as otherwise our master branch is not working anyway. 
Thanks for all contributions.

**Comment by EulalieCoevoet** [comment]
Wait

**Comment by damienmarchal** [comment]
Lol @EulalieCoevoet

**Comment by EulalieCoevoet** [comment]
xD ok I will commit directly in the master branch ^^ never mind

**Pull request merged**